templates/servermessage/404-en-fr.html

```diff
@@ -0,0 +1,90 @@
+---
+title: "Page not found / Page non trouvée"
+language: "en"
+layout: servermessage
+nositesearch: true
+breadcrumbs: false
+nomenu: true
+isBilingual: true
+---
+<h1 class="wb-inv" id="wb-cont">Page not found / <span lang="fr">Page non trouvée</span></h1>
+<div class="col-md-12 mrgn-bttm-xl">
```

Comment:
We will need to add an invisible skip link for AT users in order for them to skip to the second language block. You can see what I did in the Theme template for reference.